driver/bpf/Makefile

```diff
@@ -56,6 +56,7 @@ $(obj)/probe.o: $(src)/probe.c \
 		$(DEBUG) \
 		-D__KERNEL__ \
 		-D__BPF_TRACING__ \
+		-DKBUILD_MODNAME='"falcobpf"' \
```

Comment:
If I grep `KBUILD_MODNAME` I see:
```
driver/driver_config.h.in:31:#ifndef KBUILD_MODNAME
driver/driver_config.h.in:32:#define KBUILD_MODNAME DRIVER_NAME
```
Maybe we should have a .in file here too, and follow what's currently done for driver/Makefile.in putting `@DRIVER_NAME@` for it

Reply:
I don't think that's actually needed; current `driver_config.h`:
```
...
#include "ppm_api_version.h"

#define DRIVER_VERSION "2.0.0-449+3f5c13b-driver"

#define DRIVER_NAME "scap"

#define DRIVER_DEVICE_NAME "scap"

#define DRIVER_COMMIT "3f5c13b6827a786d802ccc540998b48fc9a97a4a"

#ifndef KBUILD_MODNAME
#define KBUILD_MODNAME DRIVER_NAME
#endif
```

KBUILD_MODNAME should be actually set with no issues to "scap".

Reply:
@alban would you mind sharing your build steps? FWIW I had some issues in the past with `driver_config.h` being in the gitignore and not noticing it wasn't the one I was expecting because of different config / reconfig attempts.